From bfa92f610a8488834c4771ecf7a7afbc53e3957a Mon Sep 17 00:00:00 2001
From: Jeegar Patel <jeegar.patel@xilinx.com>
Date: Tue, 22 Sep 2020 05:05:25 -0700
Subject: [PATCH 1622/1852] drm: xlnx: bridge: Fix UNINIT type warnings

commit cf82166d171e7f94388fd19eb5d58a6c7abc78c3 from
https://github.com/Xilinx/linux-xlnx.git

This patch fix UNINIT type coverity warnings.

Declaring variable "fmt"/"width"/"height" without initializer.

Using uninitialized value "fmt"/"width"/"height" when calling
"xlnx_bridge_set_input".

Signed-off-by: Jeegar Patel <jeegar.patel@xilinx.com>
Reviewed-by: Hyun Kwon <hyun.kwon@xilinx.com>
State: pending
Signed-off-by: Yaliang Wang <Yaliang.Wang@windriver.com>
---
 drivers/gpu/drm/xlnx/xlnx_bridge.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/drivers/gpu/drm/xlnx/xlnx_bridge.c b/drivers/gpu/drm/xlnx/xlnx_bridge.c
index 120902cf3f49..3cd0d83eac2e 100644
--- a/drivers/gpu/drm/xlnx/xlnx_bridge.c
+++ b/drivers/gpu/drm/xlnx/xlnx_bridge.c
@@ -343,7 +343,7 @@ static ssize_t xlnx_bridge_debugfs_write(struct file *f, const char __user *buf,
 	} else if (!strncmp(buf, "set_input", 3)) {
 		char *cmd, **tmp;
 		char *w, *h, *f;
-		u32 width, height, fmt;
+		u32 width = 0, height = 0, fmt = 0;
 		int ret;
 
 		cmd = kzalloc(size, GFP_KERNEL);
-- 
2.31.1

